lib/atomic/misc.less

```diff
@@ -303,12 +303,13 @@
 }
 
 //  --  Added hover styles for box-shadow
-.bs-sm.bs-hover:hover { box-shadow: 0 var(--su-static2) var(--su-static8) @black-legacy-050 !important; }
-.bs-md.bs-hover:hover { box-shadow: 0 var(--su-static4) var(--su-static8) @black-legacy-100 !important; }
-.bs-lg.bs-hover:hover { box-shadow: 0 var(--su-static4) var(--su-static12) fade(@black-legacy-300, 70%) !important; }
-.bs-i-sm.bs-hover:hover { box-shadow: inset 0 var(--su-static1) var(--su-static8) 0 fade(@black-legacy-200, 75%) !important; }
-.bs-i-md.bs-hover:hover { box-shadow: inset 0 var(--su-static1) var(--su-static8) 0 fade(@black-legacy-350, 80%) !important; }
-.bs-i-lg.bs-hover:hover { box-shadow: inset 0 var(--su-static1) var(--su-static12) 0 @black-legacy-350 !important; }
+// TODO do we still need them? Can we deprecate/remove these weird hover classes?
+.bs-sm.bs-hover:hover { box-shadow: 0 var(--su-static2) var(--su-static8) .set-black()[150] !important; }
+.bs-md.bs-hover:hover { box-shadow: 0 var(--su-static4) var(--su-static8) .set-black()[225] !important; }
+.bs-lg.bs-hover:hover { box-shadow: 0 var(--su-static4) var(--su-static12) fade(.set-black()[350], 70%) !important; }
+.bs-i-sm.bs-hover:hover { box-shadow: inset 0 var(--su-static1) var(--su-static8) 0 fade(.set-black()[300], 75%) !important; }
+.bs-i-md.bs-hover:hover { box-shadow: inset 0 var(--su-static1) var(--su-static8) 0 fade(.set-black()[350], 80%) !important; }
+.bs-i-lg.bs-hover:hover { box-shadow: inset 0 var(--su-static1) var(--su-static12) 0 .set-black()[350] !important; }
```

Comment:
A few notes here:

- I believe **none of this** is documented in Stacks. We should either a) document it or b) get rid of these classes.
- We have a few references to `.bs-hover` in Core but _I think_ they could be replaced with conditional classes (like `h:bs-sm`). I'm not sure why we have these hover classes.
- AFAICT, these `.bs-i-[*]` classes are unused in Core. I'd feel pretty comfortable getting rid of them.

Reply:
Maybe this is something to add to our famous document too. :) 
At the end of this initiative we could create few stories out of it and prioritize it at our own pace. 
We could even decide to work on some of them before cutting the 2.0 final release.

Reply:
Added to our "Color Redefinition: Breaking changes, trade offs, details" doc under "[DEPRECATION: Box-shadow atomic hover classes](https://docs.google.com/document/d/1dqIymDut_dRHblyvuMhgpxeFlA6-dh9VPAdb3kT7vSA/edit#heading=h.w6a14mf4uf1v)"